README.md

```diff
@@ -133,6 +133,8 @@ expect(valueA: any).toMatchDiffSnapshot(valueB: any, options?: Options, testName
 - `expand: boolean` (default: `false`) – expand the diff, so the whole information is preserved
 - `colors: boolean` (default: `false`) – preserve color information from Jest diff
 - `contextLines: number` (default: 5) - number of context lines to be shown at the beginning and at the end of a snapshot
+- `omitPatchMarks: boolean` (default: `false`) - prevent line number patch marks from appearing in
+diffs. This can be helpful when diffs are breaking only because of the patch marks.
```

Comment:
Would also be cool to show an example like: changes @@ -1,1 +1,2 @@ to @@ --- --- @

Reply:
done